src/page-modules/contact/utils.ts

```diff
@@ -75,6 +77,20 @@ export const setLineAndResetStops = (context: any, line: Line) => {
   };
 };
 
-export const formatLineName = (line: Line): string => {
-  return line.publicCode ? `${line.publicCode} - ${line.name}` : line.name;
+export const getLineOptions = (lines: Line[]): Option<Line>[] => {
+  return lines.map((line) => ({
+    id: line.id,
+    name: line.publicCode ? `${line.publicCode} - ${line.name}` : line.name,
+    value: line,
+  }));
+};
+
+export const getStopOptions = (
+  stops: Line['quays'],
+): Option<Line['quays'][0]>[] => {
+  return stops.map((stop) => ({
+    id: stop.id,
+    name: stop.name,
+    value: stop,
+  }));
```

Comment:
These utility functions are very specific to the new search component, so I wouldn't place it in this file but rather have it closer to the component itself

Reply:
Maybe create a folder names `searchable-selector` inside `components` containing a `utils.ts` with the functions?